core/embed/firmware/header.S

```diff
@@ -7,6 +7,9 @@
     .type g_header, %object
     .size g_header, .-g_header
 
+#if TREZOR_MODEL != 1
```

Comment:
Can we `#ifdef` only different parts of the header?

There are lots of lines which are the very same.